boarman_ryan_lidar_23.ipynb

```diff
@@ -0,0 +1,523 @@
+      "text/plain": [
+       "<Figure size 1000x500 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Create stats to enter into plot using calc_height_stats function\n",
+    "soap_height_stats = calc_height_stats(\n",
+    "    plots_path=soap_plots_path,\n",
+    "    chm_path=soap_chm_path,\n",
+    "    insitu_path=soap_insitu_path,\n",
+    "    id_col='ID')\n",
+    "\n",
+    "\n",
+    "fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(10, 5))\n",
+    "\n",
+    "# Plot 1\n",
+    "create_subplot(soap_height_stats,\n",
+    "               ax=ax1,\n",
+    "               x=\"lidar_max\",\n",
+    "               y='insitu_max',\n",
+    "               xlabel=\"Lidar Derived Max Tree Height (m)\",\n",
+    "               ylabel=\"Measured Tree Height (m)\",\n",
+    "               title=\"Lidar vs. Measured Max Tree Height \\n NEON Soap \"\n",
+    "               \"Field Site\",\n",
+    "               xlim_min=0,\n",
+    "               xlim_max=140,\n",
+    "               ylim_min=0,\n",
+    "               ylim_max=140)\n",
+    "\n",
+    "# Plot 2\n",
+    "create_subplot(soap_height_stats,\n",
+    "               ax=ax2,\n",
+    "               x=\"lidar_mean\",\n",
+    "               y='insitu_mean',\n",
+    "               xlabel=\"Lidar Derived Mean Tree Height (m)\",\n",
+    "               ylabel=\"Measured Tree Height (m)\",\n",
+    "               title=\"Lidar vs. Measured Mean Tree Height \\n NEON Soap Field Site\",\n",
+    "               xlim_min=0,\n",
+    "               xlim_max=40,\n",
+    "               ylim_min=0,\n",
+    "               ylim_max=40)\n",
+    "\n",
+    "plt.show()"
```

Comment:
Ran into an error here that says "[Errno 2] No such file or directory: 'california/neon-soap-site/2013/insitu/veg_structure/D17_2013_SOAP_vegStr.csv'" I think it should be 'veg-structure'

Reply:
yes you are correct. small typo